rai/node/node.cpp

```diff
@@ -515,6 +515,7 @@ class network_message_visitor : public rai::message_visitor
 		}
 		node.stats.inc (rai::stat::type::message, rai::stat::detail::confirm_req, rai::stat::dir::in);
 		node.peers.contacted (sender, message_a.header.version_using);
+		node.process_active (message_a.block);
```

Comment:
Both process_active and publish were removed, should both be put back ?

Reply:
I think it's safe to leave it out here from the previous discussion about it above.  Plus, then we'd have 2 more pull requests to change things around !